### PR TITLE
@orta => A few additions to analytics before release 📱

### PR DESCRIPTION
**Pull request by ashfurrow**
Specifically, adds tracking for active user signups, tracks umbrella event for all account creations, and tracks source of individual signup (twitter, fb, email). 

Also removes a few unused constants.

### CODE REVIEW

**Comment by ashfurrow** [comment]
So I, uhh, went to force-push an amended commit to this branch, but didn't realize I was on master. Fixing master first, then pushing up to here.

**Comment by orta** [comment]
use `make push` to not have these problems again: https://github.com/artsy/eigen/blob/master/Makefile#L123-L130

**Comment by ashfurrow** [comment]
:+1:

**Comment by ashfurrow** [comment]
OK this should be good now.

File: Artsy/Classes/ARAppDelegate+Analytics.m [hunk 1]
```diff
@@ -422,6 +422,39 @@ - (void)setupAnalytics
                     ]
                 },
                 @{
+                    ARAnalyticsClass: ARSignUpActiveUserViewController.class,
+                    ARAnalyticsDetails: @[
+                        @{
+                            ARAnalyticsEventName: ARAnalyticsTappedLogIn,
+                            ARAnalyticsSelectorName: @"goToLogin:",
```
Comment (orta) [inline Artsy/Classes/ARAppDelegate+Analytics.m]: `ARAnalyticsSelectorName: NSStringFromSelector(@selector(goToLogin)),`?

File: Artsy/Classes/ARAppDelegate+Analytics.m [hunk 2]
```diff
@@ -886,7 +931,24 @@ - (void)setupAnalytics
...
-                        }
+                        },
+                    ]
+                },
+                @{
+                    ARAnalyticsClass: ARSignupViewController.class,
+                    ARAnalyticsDetails: @[
+                        @{
+                            ARAnalyticsEventName: ARAnalyticsSignUpTwitter,
+                            ARAnalyticsSelectorName: @"fb:",
```
Comment (orta) [inline Artsy/Classes/ARAppDelegate+Analytics.m]: same

File: Artsy/Classes/ARAppDelegate+Analytics.m (hunk 1 above)
Comment (ashfurrow) [inline Artsy/Classes/ARAppDelegate+Analytics.m]: I'm sort of up in the air about this. The file currently swings both ways, but essentially the issue is that the selector is private, so using that, or the `ARAnalayticsSelector()` macro that wraps it, doesn't add any safety. Still, I'll add it for consistency.

**Comment by orta** [comment]
OK cool :👍

Merge on green

**Pull request merged**